### ISSUE TITLE
Regression Sample

### ISSUE THREAD

**Issue by tolgabirdal** [tiny-dnn/tiny-dnn#532]
It would be good to have some example on doing regression with tiny-dnn. The documentation briefly mentions it, but it doesn't involve giving information about the construction of the network and the number of nodes in the output units and choice of loss functions.

**Comment by edgarriba** [comment]
hi @tolgabirdal! Will be nice that someone contribute with such example :wink: 

A simple example to reproduce can be like this
https://github.com/pytorch/examples/blob/master/regression/main.py

**Comment by sshkhrnwbie** [comment]
Hi @edgarriba

I am new to tiny-dnn and would like to contribute. I believe this should be simple enough for a beginner. 

How would you prefer this be done: a single file in a separate regression directory in the main examples directory like torch or some other way?

**Comment by Randl** [comment]
@sshkhrnwbie Look at how other examples are organized:
https://github.com/tiny-dnn/tiny-dnn/tree/master/examples/mnist

I suppose you create separate folder with code and readme file.
